source/content/shibboleth-sso.md

```diff
@@ -192,7 +192,9 @@ You can now visit the subdirectory `/simplesaml` on your development site and co
 
 If using the [simpleSAMLphp Authentication](https://www.drupal.org/project/simplesamlphp_auth) module, follow the instructions listed in their [README](https://git.drupalcode.org/project/simplesamlphp_auth). These instructions cover both Composer and non-Composer implementations for Drupal 8 sites.
 
-For non-Composer implementations, you can add the following lines to `settings.php` so that the Drupal module can locate SimpleSAMLphp:
+<Alert title="Note" type="info">
+
+If using Composer, the `setting.php` is already configired to locate the SimpleSAMLphp file. For non-Composer implementations, you can add the following lines to `settings.php` so that the Drupal module can locate SimpleSAMLphp:
```

Comment:
I wouldn't say "is already configured"; it's more like "the settings.php configuration is not needed"